pokemongo_bot/cell_workers/move_to_fort_worker.py

```diff
@@ -41,8 +41,16 @@ def work(self):
                 )
 
 
-            if not self._step_walker.step():
+            step_walker = self._step_walker
+                if step_walker == True:
+                self.api.player_update(latitude=lat, longitude=lng)
```

Comment:
don't know if you saw my comment, but please move the `self.api.player_update` call to after the `StepWalker` finishes moving and leave this worker untouched.